### DISCUSSION TITLE
easycap_BC_TMS64_X21 electrode channel name

### DISCUSSION THREAD

**Discussion by xiongyy99255** [simnibs/simnibs#385] [9. Results and analysis] [answered]
Hi SimNIBS team, thanks for the great tool!
Where should I find the channel name(electrode name, like F3,F4...) corresponding to /SimNIBS4.1/resources/ElectrodeCaps_MNI/easycap_BC_TMS64_X21.csv? When I use easycap_BC_TMS64_X21 to generate leadfield, the .hdf5 file I read about the EEG channel names are all 1,2,3,4,.....

Thanks!
![easycap_BC_TMS64_X21](https://github.com/user-attachments/assets/27929ebb-37a2-4fd3-949b-af3c31beb246)
![easycap_BC_TMS64_X21_2](https://github.com/user-attachments/assets/ebc43ae3-6e05-43ec-ac54-43bd964147c0)

**Comment by jduemose** [comment, marked as answer]
Hi, this particular cap has an equidistant layout - specifically an M10 layout - so the electrode positions generally do not correspond to those of the 10% system although at some positions they do. Check out [this](https://www.easycap.de/wp-content/uploads/2018/02/Easycap-Equidistant-Layouts.pdf) document from easycap where you can see an overlay of the M10 on the 10% system.
